sidechain/consensus/aura/src/block_importer.rs

```diff
@@ -181,6 +220,23 @@ impl<Authority, PB, SB, OCallApi, StateHandler, StateKey, TopPoolExecutor> Block
 		&self.ocall_api
 	}
 
+	fn import_parentchain_block(
+		&self,
+		sidechain_block: &SB::Block,
+		last_imported_parentchain_header: &PB::Header,
+	) -> Result<PB::Header, ConsensusError> {
+		let maybe_latest_imported_block = self
+			.parentchain_block_import_trigger
+			.import_until(|signed_parentchain_block| {
+				signed_parentchain_block.block.hash() == sidechain_block.layer_one_head()
```

Comment:
This little piece of code contains alooot of complexity of why we only import up to the parentchainb lock of the latest sidechain block.
I think here a comment would help to make it easier to understand.

Proposing:
`// To prevent sidechain forks, we only sync with latest parentchain block upon slot or sidechain block (issue #423)`

Reply:
done 👍